lib/plugins/aws/provider/awsProvider.js

```diff
@@ -356,9 +356,8 @@ class AwsProvider {
           awsResourceTags: {
             type: 'object',
             patternProperties: {
-              '^(?!aws:)[\\w./=+-]{1,128}$': {
+              '^(?!aws:)[\\w./=+-:]{1,128}$': {
```

Comment:
I think we need to move `-` so it's listed last, otherwise `+-:` implies a _range_

Reply:
Thanks @medikoo. I've updated it.